examples/todomvc/app.re

```diff
@@ -6,7 +8,7 @@ let enterKey = 13;
 
 let namespace = "reason-react-todos";
 
-let saveLocally todos => ReasonJs.LocalStorage.setItem namespace (ReasonJs.JSON.stringify todos);
+let saveLocally todos => ReasonJs.LocalStorage.setItem namespace (ReasonJs.Json.stringify todos);
```

Comment:
Not at all. The naming scheme for abbreviations was one of the open questions on the reason-js PR. This naming convention was more consistent when the files were stand-alone modules with names generated from camelCase filenames. A lot of names are affected by this naming convention, "Html", "Dom" etc., and I'd personally prefer all upper case for capitalized names, and all lower case for uncapitalized names if convention allows it.